### DISCUSSION TITLE
umijs/max使用 configProvider.theme.hashed:false无法关闭 dev 样式

### DISCUSSION THREAD

**Discussion by Mankvis** [umijs/umi#12483] [Q&A]
想关闭如下dev 样式
![image](https://github.com/umijs/umi/assets/49258868/f460c0df-3eda-4922-b1df-836a87bbaf21)
在其他地方找到解决方式如下
![image](https://github.com/umijs/umi/assets/49258868/815d4076-bbcc-4702-998e-314e7b240fa6)
但是在配置文件中设置在 umijs 中没有效果
![image](https://github.com/umijs/umi/assets/49258868/a7b2f16f-7b47-4e95-a763-e6658cde88e5)

**Comment by fz6m** [comment]
使用 `antd.theme` ：

```ts
// .umirc.ts

  antd: {
    theme: {
      hashed: false
    }
  }
```

**Reply by Mankvis** [reply]
> 使用 `antd.theme` ：
> 
> ```ts
> // .umirc.ts
> 
>   antd: {
>     theme: {
>       hashed: false
>     }
>   }
> ```

我并没有使用 `.umirc.ts`，而是使用了 `config.ts`，以下是内容，不过并没有效果

```js
export default defineConfig({
  antd: {
    configProvider: {},
    appConfig: {},
    theme: {
      hashed: false,
    },
  },
  access: {},
  model: {},
  initialState: {},
  request: {},
  layout: {
    title: '@umijs/max',
  },
  routes,
  npmClient: 'pnpm',
  dva: {},
});
```

**Reply by fz6m** [reply]
如有问题，给一个最小复现仓库看看吧。
